test/basic.html

```diff
@@ -16,16 +16,16 @@
   <script src="../../webcomponentsjs/webcomponents.js"></script>
   <script src="../../web-component-tester/browser.js"></script>
 
-  <link rel="import" href="../core-range.html">
+  <link rel="import" href="../iron-range.html">
 
 </head>
 <body>
 
-  <core-range></core-range>
+  <iron-range></iron-range>
```

Comment:
Consider using [`<test-fixture>`](https://github.com/PolymerElements/test-fixture) so that state doesn't leak across tests.